js/ui/grid_core/ui.grid_core.column_chooser.js

```diff
@@ -454,6 +461,16 @@ const columnChooserMembers = {
         return popupContainer && popupContainer.option('visible');
     },
 
+    isSelectMode: function() {
+        return this.option('columnChooser.mode') === 'select';
+    },
+
+    hasHiddenColumns: function() {
+        const isEnabled = this.option('columnChooser.enabled');
+
+        return isEnabled && !!this.getColumns().filter(column => !column.visible).length;
```

Comment:
Minor comment.

Let's split this complex condition into a separate variable:
```suggestion
       const hasHiddenColumns = !!this.getColumns().filter(column => !column.visible).length;

        return isEnabled && hasHiddenColumns;
```